fury/actor.py

```diff
@@ -1451,6 +1451,99 @@ def sphere(centers, colors, radii=1., theta=16, phi=16,
     return actor
 
 
+def cone(centers, directions, colors, heights=1., resolutions=10,
+         vertices=None, faces=None):
+    """Visualize one or many cones with different colors and radii
+
+    Parameters
+    ----------
+    centers : ndarray, shape (N, 3)
+    colors : ndarray (N,3) or (N, 4) or tuple (3,) or tuple (4,)
+        RGB or RGBA (for opacity) R, G, B and A should be at the range [0, 1]
+    radii : float or ndarray, shape (N,)
+    theta : int
+    phi : int
```

Comment:
Can you update the documentation? It missed `directions`, `height`, `resolutions` and some parameters needs to be removed.